app/test_main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+from typing import Callable
+from unittest.mock import patch
+from app.main import can_access_google_page
+
+
+url = "http://www.google.com"
+
+
+@patch("app.main.valid_google_url")
+@patch("app.main.has_internet_connection")
+def test_is_accessible(mock_internet: Callable, mock_url: Callable) -> None:
+    mock_internet.return_value = True
+    mock_url.return_value = True
+    assert can_access_google_page(url) == "Accessible"
```

Comment:
Add an error message. It's good when the user understands what the problem is. Fix everywhere

Reply:
what about this test?